guide/sapphire/first-command.md

```diff
@@ -0,0 +1,80 @@
+# Your first command
+
+Now that you have created the `commands` folder, you're ready to make your first command file! In Sapphire, we usually define one command per file, but you can also define groups of them. Hop over to your `commands` folder, and create a new file, we'll show how to make a ping command, so we will name this `ping.js`. This is going to be a simple command that replies to a message and then edits with the elapsed time. We'll go into arguments and other features later.
+
+Once you have your file, it's time to get started!
+
+## Creating your command class
+
+Before you do anything, at the start of your file, you're going to need to require Sapphire Framework again, specifically its Command class.
+
+```js
+const { Command } = require('@sapphire/framework');
+```
+
+Commands are classes that are exported in any of the following ways:
+
+```js
```

Comment:
Maybe this can have only commonjs as most parts of guide follow common js and having two of them *might* confuse the user on which to use